doc/cug.tex

```diff
@@ -3447,7 +3447,9 @@ \subsubsection{Trigger Types}
 suite (you can run the {\em AutoRecover.async} example suite to see how
 this works).  The dashed graph edges ending in solid dots indicate
 suicide triggers, and the open arrowheads indicate conditional triggers
-as usual.
+as usual. Note as default the suicide triggers are ignored in the graphing. 
+Under {\em View}, {\em Options} you can select to toggle 
+{\em Ignore Suicide Triggers}.
```

Comment:
I'm being pedantic sorry, but I "as default" and "ignored in the graphing" don't sound right to me.  I'd prefer something like *Suicide triggers are ignored by default in the graph view, unless you toggle them on with View -> Options -> Ignore Suicide Triggers*.